src/services/ResourceDirectManager.js

```diff
@@ -43,6 +44,17 @@ async function assignRole (legacyChallengeId, roleId, userId, handle, copilotPay
     }
   }
 
+  const compInfo = await RegistrationDAO.registerComponentInquiry(userId, legacyChallengeId)
+  if (compInfo.projectCategoryId === DEVELOPMENT_PROJECT_TYPE) {
```

Comment:
this is not correct way to check.

Reply:
@eisbilir what's the right way to do it? That's how it's done in the legacy API

Reply:
legacy api, checks if project_studio_spec_id existst or not to decide if it is studio challenge. All other challenges considered as Development type.
Instead of checking project_studio_spec_id, we can look at v5 data to decide if it is a studio challenge

Reply:
compInfo.projectCategoryId === DEVELOPMENT_PROJECT_TYPE
this should be replaced with !isStudioChallenge